src/game/Cbuf/Cbuf.cpp

```diff
@@ -53,4 +53,38 @@ int Conbuf_CleanText(const char* source, char* target, int sizeofTarget)
 
 	*target = 0;
 	return target - start;
+}
+
+//THUNK : 0x004F5770
+long Conbuf_AppendText(const char* a1)
+{
+	char* v1;
+	std::int32_t v2;
+	char lParam[32768];
+
+	if (strlen(a1) <= 16383)
+	{
+		v1 = (char*)a1;
+	}
+	else
+	{
+		v1 = (char*)&a1[strlen(a1) - 16383];
+	}
+
+	v2 = (std::int32_t)memory::call<unsigned char*(char*, int, char*)>(0x64DD30)(lParam, 0x8000, v1);
```

Comment:
the memory call goes to `Conbuf_CleanText`, which we wrote

Reply:
My IDA instance doesn't always apply symbols from Lumina and I don't know why, otherwise I would have labeled it

Reply:
me neither, so I just compare it to T6 Dedi IDB. Sys_* code doesn't seem to change that much between CODs so a good guess can be made